jobs/integr8ly/sso-user-create.yaml

```diff
@@ -0,0 +1,67 @@
+---
+
+- job:
+    name: sso-user-create-tests
+    project-type: pipeline
+    sandbox: true
+    parameters:
+      - string:
+          name: REPOSITORY
+          default: 'https://gitlab.cee.redhat.com/integreatly-qe/integreatly-qe.git'
+          description: 'QE Git repository URL.'
+      - string:
+          name: BRANCH
+          default: 'master'
+          description: 'Branch of the repository'
+      - string:
+          name: SSO_URL
+          description: 'SSO URL for the cluster on which the tests will be executed.'
+      - bool:
+          name: CLEAN_RESOURCES_USERS
+          default: true
+          description: 'Whether the users should be deleted after the successful execution or not.'
+      - string:
+          name: NUMBER_OF_USERS
+          default: "5"
```

Comment:
one minor thing - I think we should make the default number consistent with the test code (e.g. 9). Other than that LGTM

Reply:
OK so I would update the test source code then since 9 seems to me too many users.:)